src/i18n/i18nFactory.ts

```diff
@@ -21,10 +22,31 @@ const translationsFiles: { [langCode: string]: any } = {
   zh: zhTranslation.translation,
 }
 
+const format = (value: any, format: string) => {
+  if (format === 'bulletList' && Array.isArray(value)) {
+    // Bullet list expected format: [{ name: 'name/translation.key', suffixes?: ['suffix', 'translation.key'] }]
+    const bulletstr = '•'
+    const _value = value.map((item: any) => {
+      const key = `$t(${item.key})`
+      const suffix = item.suffixes ? `(${item.suffixes.map((suffix: any) => `$t(${suffix})`).join(', ')})` : ''
```

Comment:
Allow multiple suffixes, this is useful for eg. Original Data Point where we want to show both year and where it is used (now only in References)
Later use cases can be variableName, colName....

Reply:
unfortunately no, and i know this...

what do you suggest?